### DISCUSSION TITLE
File path relative to current note

### DISCUSSION THREAD

**Discussion by christophberger** [coddingtonbear/obsidian-csv-table#23] [Ideas]
Hi, 

Thanks for creating and maintaining this much-needed plugin. Unfortunately, I am unable to use it because of the requirement that the source path must be relative to the vault's root. (In other words, it's an absolute path from the vault's perspective.)

I would rather like to set the path relative to the current note.
For example, if my note's attachments reside in folder `files` below the current folder, the `source:` value should be `files/table.csv` (or maybe `./files/table.csv`, to distinguish it from paths relative to the vault's root).

Reason: 

With a path relative to the vault's root, if I move the note and its attachment folder to another folder, the `source` path breaks. 

A path relative to the current note would survive any note rearrangements, as long as the relative path to the note's attachments does not change.

**Comment by vbrozik** [comment]
This were exactly my thought when I started testing the plugin. I was wondering why the plugin does not work then I realized that the csv file path is "relative" to the vault's root.

IMHO this is contrary to the Obsidian conventions and I think the path should be truly relative to the markdown file directory by default.

Related discussion: https://github.com/coddingtonbear/obsidian-csv-table/discussions/32 use relative path for csv file
